src/SeqCli/Cli/Commands/ApiKey/ListCommand.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using SeqCli.Cli.Features;
+using SeqCli.Connection;
+using Serilog;
+
+namespace SeqCli.Cli.Commands.ApiKey
+{
+    [Command("apikey", "list", "Send a structured log event to the server", Example =
+        "seqcli log -m 'Hello, {Name}!' -p Name=World -p App=Test")]
+    class ListCommand : Command
+    {
+        private readonly SeqConnectionFactory _connectionFactory;
+        private readonly ConnectionFeature _connection;
+
+        public ListCommand(SeqConnectionFactory connectionFactory)
+        {
+            _connectionFactory = connectionFactory;
+            _connection = Enable<ConnectionFeature>();
+        }
+
+        protected override async Task<int> Run()
+        {
+            var connection = _connectionFactory.Connect(_connection);
+
+            var apiKeys = await connection.ApiKeys.ListAsync();
+            Log.Debug("Retrieved ApiKeys {@ApiKeys}", apiKeys);
+            var data = apiKeys.Select(a => new
+            {
+                a.AppliedProperties,

```

Comment:
If we listed `Id` and `Title` first here, it might be a bit quicker to visually parse the output.